apps/backoffice-v2/src/common/components/molecules/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, ComponentProps } from 'react';
+import { CalendarIcon } from '@radix-ui/react-icons';
+import { formatDate } from '@/common/utils/format-date';
+import { DateRange } from 'react-day-picker';
+import dayjs from 'dayjs';
+import { ctw } from '@/common/utils/ctw/ctw';
+import { Button } from '../../atoms/Button/Button';
+import { Calendar } from '../../organisms/Calendar/Calendar';
+import { Popover, PopoverContent, PopoverTrigger } from '@ballerine/ui';
+
+type TDateRangePickerProps = {
+  onChange: ComponentProps<typeof Calendar>['onSelect'];
+  value: ComponentProps<typeof Calendar>['value'];
+  className: ComponentProps<'div'>['className'];
+};
+
+export function DateRangePicker({ onChange, value, className }: TDateRangePickerProps) {
+  const currentDate = new Date();
+  const [date, setDate] = useState<DateRange | undefined>({
+    from: new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate()),
+    to: dayjs(new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate(), 1))
+      .add(1, 'month')
+      .toDate(),
+  });
+
+  const handleDateChange = (selection: DateRange | undefined) => {
+    setDate(selection);
+    onChange(selection);
+  };
```

Comment:
I was expecting these lines to be removed and for the component to receive onChange and value as props. 🙂
```suggestion
```

Reply:
Hey, what if we only receive the onChange as prop, not the value and className?